docs/security.md

```diff
@@ -3,96 +3,12 @@ post_title: Security
 menu_order: 40
 enterprise: 'no'
 ---
-# Mesos Security
 
-## SSL
-
-<table class="table">
```

Comment:
Yes, I think all docs should live with the source code for the project it's documenting.  Otherwise, the docs and product get out of sync very quickly.